### ISSUE TITLE
Resource form misses relation to a project

### ISSUE THREAD

**Issue by wenzeslaus** [OSGeo/osgeo#17]
The Resource form mentions relation to event or committee, but it never explicitly mentions relation to the software project. I was transferring http://www.osgeo.org/node/1592 there and I missed filed for related OSGeo (software) projects, other open source software, and any other software. What I did I put GRASS GIS to tags but that seemed not sufficient.

**Comment by jodygarnett** [comment]
Good point, any suggestions on how to surface that? Would you expect it to be listed on the project page for example ...

If not search is a good way to locate grass resources.
